sections/combat.tex

```diff
@@ -195,7 +195,7 @@ \subsection*{\hypertarget{Combatexperience}{Combat Experience}}
 
 \subsection*{\hypertarget{Quick}{Quick Combat}}\index{Quick Combat}
 If your Hero's Level is higher than a Field's Difficulty when Combat against Neutral Units would begin, \textbf{no Combat} takes place.
-The player is considered to have beaten the Neutral Units by default and gains no rewards from the Combat.
+The player is considered to have beaten the Neutral Units by default and gains no rewards from the Combat itself, then proceeds to Visit the Field.
```

Comment:
Why would the player not visit the field after combat? It's unnecessary.

Reply:
The previous sentence literally says "gains no rewards". It *obviously* means that you get no benefits from the chest, from the mine, from the settlement, etc. Right?